cli/cmd/compliance_test.go

```diff
@@ -237,3 +238,26 @@ var (
 		Violations:            []api.ComplianceViolation{},
 	}
 )
+
+func TestRecommendationIDRegex(t *testing.T) {
+	ok, _ := regexp.MatchString(RecommendationIDRegex, "invalid")
+	assert.False(t, ok, "recommendation id must be uppercase")
+
+	ok, _ = regexp.MatchString(RecommendationIDRegex, "")
+	assert.False(t, ok, "recommendation id cannot be empty string")
+
+	ok, _ = regexp.MatchString(RecommendationIDRegex, "44LW_AWS_ELASTICSEARCH_3")
+	assert.False(t, ok, "recommendation id cannot be start with a number")
+
+	ok, _ = regexp.MatchString(RecommendationIDRegex, "_LW")
+	assert.False(t, ok, "recommendation id cannot start with underscore")
+
+	ok, _ = regexp.MatchString(RecommendationIDRegex, "LW_AWS_ELASTICSEARCH_3")
+	assert.True(t, ok, "recommendation id must start with uppercase letter, may contain underscores and numbers")
+
+	ok, _ = regexp.MatchString(RecommendationIDRegex, "LW_AWS_NETWORKING_46")
+	assert.True(t, ok, "recommendation id must start with uppercase letter, may contain underscores and numbers")
+
+	ok, _ = regexp.MatchString(RecommendationIDRegex, "AWS_CIS_3_3")
+	assert.True(t, ok, "recommendation id must start with uppercase letter, may contain underscores and numbers")
+}
```

Comment:
Can we do Table Driven Tests here, we have done it in other parts of the codebase.

-- https://medium.com/@pliutau/table-driven-tests-in-go-5d7e230681da